lightspeed/modules/proc_generate-training-data-set.adoc 

```diff
@@ -0,0 +1,102 @@
+:_content-type: PROCEDURE
+
+[id="generate-training-data-set_{context}"]
+
+= Generating a custom model training data set
+
+After installing {AnsibleContentParser}, run it to scan your custom Ansible files and generate a custom model training data set. The training data set includes a list of Ansible files and their paths relative to the project root. You can then upload the  training data set to {ibmwatsonxcodeassistant} and create a custom model for your organization. If you used ansible-lint preprocessing and encountered rule violations, you must resolve the rule violations (URL) before uploading the training data set to {ibmwatsonxcodeassistant}.
```

Comment:
Looks like a double space here: "then upload the  training data..."

Reply:
Also: "you must resolve the rule violations (URL)" 
I presume (URL) may be a placeholder, in which case disregard this flag :)

Reply:
#1 is done. #2: yes, I'll update the xrefs shortly.